pulseapi/entries/views.py

```diff
@@ -47,6 +47,13 @@ def post_validate(request):
 
     return True
 
+class EntriesPagination(PageNumberPagination):
+    """
+    Add support for pagination and custom page size
+    """
+    page_size = 48
+    page_size_query_param = 'page_size'
```

Comment:
Turns out if you don't include this, you can't control it. http://www.django-rest-framework.org/api-guide/pagination/#configuration